core/src/test/java/org/apache/accumulo/core/client/ZooKeeperInstanceTest.java

```diff
@@ -22,7 +22,8 @@
 import static org.easymock.EasyMock.createMock;
 import static org.easymock.EasyMock.expect;
 import static org.easymock.EasyMock.replay;
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
It would be very useful if all of our uses of `assert*()` methods were already using static imports. If that were done already in our existing code, it would make this migration go more smoothly, because only the import statement would change, which makes reviewing easier. We may already be using static imports for these everywhere, though.

Reply:
I think it may already be the case that all `assert*()` methods are static imports

Reply:
You are right. It looks like I added some checkstyle rules for that awhile back. These rules will need to be updated at some point before this migration is fully done:

https://github.com/apache/accumulo/blob/912292d2131dbaf0f9cc28aff464608b478bf362/pom.xml#L1084-L1091